rust/operator-binary/src/main.rs

```diff
@@ -1,6 +1,8 @@
 mod config;
 mod controller;
 
+use crate::controller::{CONTROLLER_NAME, OPERATOR_NAME};
```

Comment:
For me `OPERATOR_NAME` can reside even in main.rs? `CONTROLLER_NAME` should reside in the respective controller (does not matter here with only one controller, but i like the consistency :))

Reply:
Done.